### DISCUSSION TITLE
Training Issue: High Force Error

### DISCUSSION THREAD

**Discussion by mcourtiel** [deepmodeling/deepmd-kit#4468] [Q&A]
Hello everyone,

I am training a model on a non periodic system (Al)45+ and  I am working with deepmd-kit V2.2.10.

The problem I am facing is that even if the error on the energy per atom is reaching 8.5e-3, the error on the forces it is "stuck" at 1.5e-1.

Here you can find an input file : 
[input.json](https://github.com/user-attachments/files/18096461/input.json)
And here the lcurve.out : 
[lcurve.txt](https://github.com/user-attachments/files/18096484/lcurve.txt)

If anyone has encountered this issue or has insights into potential solutions, I would greatly appreciate your advice.

Thanks a lot !

**Comment by wanghan-iapcm** [comment]
What is the standard deviation of the force labels?

**Reply by mcourtiel** [reply]
The standard deviation on forces is 0.49924 eV/A.

**Comment by njzjz** [comment]
Could you plot some figures like #3678?
